src/types/pass.d.ts

```diff
@@ -0,0 +1,61 @@
+export interface IgetPassFunction {
+  take: number | undefined;
+  currentPage?: number;
+  targetPage?: number;
+  firstItemId?: number;
+  lastItemId?: number;
+  passStatusOptions: 'AVAILABLE' | 'DISABLED';
+  filterOption: 'LATEST' | 'HIGHEST_PRICE' | 'BEST_SELLING';
+  lectureId?: string | number;
+}
+
+export interface IresponsePassData {
+  itemList: IpassData[];
+  totalItemCount: number;
+}
+
+export interface IpassData {
+  id: number;
+  title: string;
+  price: number;
+  availableMonths: number;
+  maxUsageCount: number;
+  salesCount: number;
+  lecturePassTarget: {
+    lecture: {
+      id: number;
+      title: string;
+    };
+  }[];
+}
+
+export interface IcreatePass {
+  title: string;
+  lectureIds: Array<{
+    label: string;
+    value: number;
+  }>;
```

Comment:
```suggestion
  lectureIds: {
    label: string;
    value: number;
  }[];
```
다른 곳에서는 위와 같은 방식으로 했는데 여기에서만 Array를 쓴 이유가 있나욤,,??

Reply:
아무 이유 없어  다시 수정할게👍